### PR TITLE
Fix Style/Alias offenses

### PR DESCRIPTION
**Pull request by magni-**
From #752.

### CODE REVIEW

File: lib/paper_trail/has_paper_trail.rb [hunk 1]
```diff
@@ -252,6 +252,7 @@ def without_versioning(method = nil)
 
       # Utility method for reifying. Anything executed inside the block will
       # appear like a new record.
+      # rubocop: disable Style/Alias
```
Comment (magni-) [inline lib/paper_trail/has_paper_trail.rb]: Reposting my original comment here:

> But the Alias false positive is an issue with Rubocop not parsing the instance_eval blocks properly. Making Rubocop happy for that case would actually end up breaking things.

**Comment by magni-** [comment]
Not quite sure what is wrong with the build (the only failure is 2.3.0/AR4/Postgres), since everything passes locally with that configuration.

File: lib/paper_trail/has_paper_trail.rb (hunk 1 above)
Comment (magni-) [inline lib/paper_trail/has_paper_trail.rb]: I submitted a fix to Rubocop here: https://github.com/bbatsov/rubocop/pull/3004

**Comment by jaredbeck** [comment]
Please rebase.

**Comment by jaredbeck** [comment]
Looks good.

All the new `alias`es are in the same lexical scope as the methods they alias, so if I understand the style guide (https://github.com/bbatsov/ruby-style-guide#alias-method-lexically) we're good there.

Dropping `track_associations` (without the question mark) could be considered a breaking change, but I doubt anyone's using it and our next release is a major version (5), so I guess it's OK as long as we add a line to the changelog about it.

I don't like inline linter comments, but these are temporary so I can live with it.

Thanks Paul!

**Pull request merged**